protocols/upnp/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.6.0
```

Comment:
Looking at the current code, couldnt do a patch bump given the lack of breaking changes?

Reply:
This PR is part of a 3 fixes as mentioned [here](https://github.com/libp2p/rust-libp2p/pull/6121#issuecomment-3173166752). I just did this to avoid some conflicts, please lemme know if this is not desirable.

Reply:
Hey! That is fine then, I will modify the 2 fixes as patches and the change to the event as a bump! 

Thank you!